pkg/ddevapp/config_test.go

```diff
@@ -214,6 +214,9 @@ func TestConfigCommandInteractiveCreateDocrootDenied(t *testing.T) {
 	// Set up tests and give ourselves a working directory.
 	assert := asrt.New(t)
 
+	err := os.Unsetenv("DRUD_NONINTERACTIVE")
```

Comment:
AH! This is the cause of our trouble with root-owned .ddev/Test*. Need to not do this!

Reply:
Good call - note that a util.Confirm() was added (to reduce code duplication and standardize interactions) that respects the DRUD_NONINTERACTIVE env var:

https://github.com/drud/ddev/pull/1241/files#diff-427f16bfc90f69b5bd53189bb655137bR41

So for now, I've updated this to unset and reset the variable after this test to allow the interaction output to be captured.